docs/code_of_conduct.md

```diff
@@ -0,0 +1,14 @@
+# The Systemd Community Conduct Guidelines
+
+This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to systemd. It applies to all “collaborative spaces”, which is defined as community communications channels (such as mailing lists, submitted patches, commit comments, etc.).
+
+- Participants will be tolerant of opposing views.
+- Participants must ensure that their language and actions are free of personal attacks and disparaging personal remarks.
+- When interpreting the words and actions of others, participants should always assume good intentions.
+- Behaviour which can be reasonably considered harassment will not be tolerated.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at systemd-conduct@googlegroups.com. This team currently consists of David Strauss <<david@davidstrauss.net>>, Ekaterina Gerasimova (Kat) <<Kittykat3756@gmail.com>>, and Zbigniew Jędrzejewski-Szmek <<zbyszek@in.waw.pl>>. It is also possible to contact any member directly.
```

Comment:
It's unclear what should be done if someone wants to report a violation about a person who receives those reports, which is unlikely given that only @keszybz regularly shows up on GitHub. Nevertheless, I think it's worth addressing.

Reply:
The assumption was that the individual email addresses would help in this case: the reporter could pick who he wants to report this to.

Reply:
Maybe, it would be helpful to explicitly say that "In the unfortunate event that you wish to make a complaint against some member, you may instead contact any of the other member(s) individually." I borrowed this from https://wiki.postgresql.org/wiki/Code_of_Conduct#Reporting.

Reply:
Yes, I think that the intent is pretty clear in this case. If you have a suggestion for better wording, that'd be great. I wasn't very happy with this version, but I couldn't come up with something better but still fairly brief.

Reply:
Ah, I'll submit a PR with your suggested text.